internal/security/secretstore/init.go

```diff
@@ -363,23 +363,23 @@ func (b *Bootstrap) BootstrapHandler(ctx context.Context, _ *sync.WaitGroup, _ s
 	// Redis 5.x only supports a single shared password. When Redis 6 is released, this can be updated
 	// to a per service password.
 
-	redis5Pair, err := getDBCredential("security-bootstrapper-redis", cred, "redisdb")
+	redisPair, err := getDBCredential("security-bootstrapper-redis", cred, "redisdb")
```

Comment:
Since we are changing the name...
```suggestion
	redisCredentials, err := getDBCredential("security-bootstrapper-redis", cred, "redisdb")
```